.circleci/config.yml

```diff
@@ -151,13 +151,17 @@ jobs:
 
     composer_lock_updater:
         <<: *defaults
+        docker:
+            - image: quay.io/pantheon-public/build-tools-ci:4.x
         steps:
             - checkout
 
             - run:
                 name: setup-environment-vars
                 command: /build-tools-ci/scripts/set-environment
 
+            - run: source $BASH_ENV
```

Comment:
Every CircleCI phase runs in its own shell, so I don't think that the `source $BASH_ENV` here has any effect on later phases.

Also, I believe that CircleCI automatically runs `source $BASH_ENV` at the beginning of each `run` line.